app/keyboard.py

```diff
@@ -0,0 +1,112 @@
+from telegram import InlineKeyboardButton, \
+    InlineKeyboardMarkup, ParseMode, KeyboardButton, ReplyKeyboardMarkup, ReplyKeyboardRemove
+
+"""
+This file will create button structures that we will use
+"""
+
+"""
+Buttons from the bottom menu
+"""
+BUTTON1_ADD_DEBT = "ADD DEBT"
+BUTTON2_PAY_DEBT = "PAY DEBT"
+BUTTON3_LIST_DEBT = "LIST OF DEBTS"
+BUTTON4_INFO = "INFORMATION"
+
+"""
+Menu buttons that will be in the chat
+"""
+
+CALLBACK_BUTTON1 = "callback_button1"
+CALLBACK_BUTTON2 = "callback_button2"
+CALLBACK_BUTTON3 = "callback_button3"
+CALLBACK_BUTTON4 = "callback_button4"
+CALLBACK_BUTTON5_BACK = "callback_button5_back"
+CALLBACK_BUTTON6 = "callback_button6"
+CALLBACK_BUTTON7 = "callback_button7"
+CALLBACK_BUTTON8 = "callback_button8"
+CALLBACK_BUTTON9 = "callback_button9"
+CALLBACK_BUTTON10 = "callback_button10"
+CALLBACK_BUTTON11 = "callback_button11"
+
+
+TITLES = {
+    CALLBACK_BUTTON1: "Add Debtor 🔥",
+    CALLBACK_BUTTON2: "List of Debtors 📝",
+    CALLBACK_BUTTON3: " Debtor 1",
+    CALLBACK_BUTTON4: "Debtor 2",
+    CALLBACK_BUTTON5_BACK: "Back 🔙",
+    CALLBACK_BUTTON6: "Debtor 3",
+    CALLBACK_BUTTON7: "Debtor 4",
+    CALLBACK_BUTTON8: "Debtor 5",
+    CALLBACK_BUTTON9: "Debtor 6",
+    CALLBACK_BUTTON10: "Find/Add Contact",
+    CALLBACK_BUTTON11: "List of Contacts"
+
```

Comment:
Please try to avoid such blank lines, all the blank lines should be meaningful

Reply:
will be fixed in the next commit

Reply:
Fixed it